packages/stark-core/package.json

```diff
@@ -64,17 +56,18 @@
     "@angular/compiler-cli": "5.x",
     "@angular/core": "5.x",
     "@angular/platform-browser": "5.x",
-    "@angular/platform-browser-dynamic": "5.x"
+    "@angular/platform-browser-dynamic": "5.x",
+    "rxjs": "5.x"
```

Comment:
angular/router is also needed in Stark Core, could you please add it?

Reply:
True, I forgot this. It's added now